api/client/stats.go

```diff
@@ -117,6 +119,48 @@ func (s *containerStats) Collect(cli *DockerCli, streamStats bool) {
 	}
 }
 
+// CalculateTotal adds up all containers' resource statistics
+func (s *stats) CalculateTotal() {
+	// get total memory
+	var totalMemory int64
+	meminfo, err := system.ReadMemInfo()
```

Comment:
This shouldn't be there though, in the client side, `system.ReadMemInfo` will return the total memory of the client system, which might not be the same as the server.